packages/frontend/src/components/PinnedItemsPanel/index.tsx

```diff
@@ -32,7 +52,17 @@ const PinnedItemsPanel: FC<Props> = ({ projectUuid }) => {
             defaultSort={{ updatedAt: SortDirection.DESC }}
             defaultColumnVisibility={{ space: false }}
             showCount={false}
-            headerTitle="Pinned items"
+            headerTitle={headerTitle}
+            headerIcon={
+                <IconInfoCircle
+                    color={Colors.GRAY5}
+                    size={17}
+                    style={{
+                        marginTop: '7px',
```

Comment:
can you use flex to align header title and header icon. I see that both are centered so 7px looks hardcoded to me